Reflection_Engine/Query/PropertyValue.cs

```diff
@@ -39,31 +39,57 @@ public static partial class Query
         [Output("value", "value of the property")]
         public static object PropertyValue(this object obj, string propName)
         {
+            string[] props = propName.Split('.');
+            foreach (string prop in props)
+            {
+                obj = obj.GetPropertyValue(prop);
+                if (obj == null)
+                    break;
+            }
+
+            return obj;
+        }
+
+        /***************************************************/
+
+        public static object GetPropertyValue(this object obj, string propName)
```

Comment:
The name of this method is not compliant, especially since it is public.  

The way you need to do this is having `PropertyValue` calling itself on the parts of the split if `propName` is containing a dot. That way you have a single method with a valid name.

Reply:
Thanks @adecler, that'd be solved.